### DISCUSSION TITLE
I tried to Install Pack but always got this error 'got an unexpected keyword argument 'use_python3'

### DISCUSSION THREAD

**Discussion by mgazzali** [StackStorm/st2#5791] [Packs] [answered]
Hello all,

I just install stackstorm 3.7 using K8 HA configuration. Just when I tried to install this pack https://github.com/StackStorm-Exchange/stackstorm-python3_test the virtual environment is failing

It mention below error 

id: 635d376c010dcbd430b1f89d
action.ref: packs.setup_virtualenv
context.user: gar\mgazzali
parameters: 
  packs:
  - python3_test
status: failed
start_timestamp: Sat, 29 Oct 2022 14:23:40 UTC
end_timestamp: Sat, 29 Oct 2022 14:23:47 UTC
result: 
  exit_code: 1
  result: None
  stderr: "st2.actions.python.SetupVirtualEnvironmentAction: DEBUG    Using proxy http://proxy-chain.intel.com:911/
Traceback (most recent call last):
  File "/opt/stackstorm/st2/lib/python3.8/site-packages/python_runner/python_action_wrapper.py", line 395, in <module>
    obj.run()
  File "/opt/stackstorm/st2/lib/python3.8/site-packages/python_runner/python_action_wrapper.py", line 214, in run
   output = action.run(**self._parameters)
  File "/opt/stackstorm/packs/packs/actions/pack_mgmt/setup_virtualenv.py", line 86, in run
    setup_pack_virtualenv(pack_name=pack_name, update=update, logger=self.logger,
**TypeError: setup_pack_virtualenv() got an unexpected keyword argument 'use_python3'**
"

would really appreciate for any help

**Comment by amanda11** [comment, marked as answer]
The use_python3 option is not available in StackStorm 3.7, that is an old option and shouldn't be being passed to the pack install.
What are you using to try and install the pack? Are you using the st2-pack-install command? 
I'd check that whatever is initiating the install, that you check that its a ST2 3.7 version and not an earlier one, and there should be no need to pass the use_python3 option - as StackStorm 3.7 only supports python3 now.

**Reply by mgazzali** [reply]
Thanks for the recommendation. I found out that my workload is referring to old PVC that used by Stackstorm 3.1. I create a new PVC and this solve the issue.
